packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -42,9 +42,10 @@ export async function getShareLink(
     logger?: ITelemetryLogger,
     scope: "anonymous" | "organization" | "default" | "existingAccess" = "existingAccess",
     type: "view" | "edit" = "edit",
+    usePPE?: boolean,
```

Comment:
Nit: I think usePPE = false is better - only two states down the road, not 3.

Reply:
And that will allow make it non-optional for functions called from here

Reply:
Hi @vladsud ,

Thanks so much for your feedback!  I had a discussion with @AndreiZe offline about this, his opinion was that this was too heavyweight.  I'm now suggesting this as an alternative:
https://github.com/microsoft/FluidFramework/pull/4349

Thanks so much,
Cooper